### ISSUE TITLE
Test suite missing files

### ISSUE THREAD

**Issue by snoyberg** [kazu-yamamoto/http2#4]
```
Failures:

  test-hpack/HPACKSpec.hs:48: (best-effort)
  1) HPACK.decodeRequestHeader decodes headers in request
       uncaught exception: IOException of type NoSuchThing (test-hpack/hpack-test-case: getDirectoryContents: does not exist (No such file or directory))
```

**Comment by kazu-yamamoto** [comment]
I think you need to do:

- (cd test-frame; sh ./prepare.sh)
- (cd test-hpack; sh ./prepare.sh)

before testing.

**Comment by snoyberg** [comment]
I'm just running this as part of my Stackage test suites, which run a standard `cabal test`. I think the extra files should be include in the tarball. If not, I'll continue leaving the http2 test suite disabled for now.

**Comment by kazu-yamamoto** [comment]
What is the purpose of testing on Stackage? The http2 library is being tested on Travis CI with three GHC versions. If testing on Stackage is important, I will add the test files.

**Comment by snoyberg** [comment]
Running the test suites via Stackage will let you know if something changed in a dependency which breaks http2 in some way.

**Comment by clinty** [comment]
We'd also like to run the test suites in Debian.

**Comment by kazu-yamamoto** [comment]
OK. I have add test case files and released a new version (1.4.2).

@snoyberg You can enable test for http2 on Stackage again.
